android/build.gradle

```
@@ -1,50 +1,69 @@
+group 'com.example.posthog_flutter'
+version '1.0-SNAPSHOT'
+
 buildscript {
+    ext.kotlin_version = '1.7.10'
```

```suggestion
    ext.kotlin_version = '1.8.10'
```

To match the Android SDK